src/main/java/AppConfig.java

```diff
@@ -0,0 +1,46 @@
+import bank.*;
+import bank.clerk.*;
+import person.Person;
+
+public class AppConfig {
+
+    private final CentralBank centralBank = new CentralBank();
+    private final BankStorage bankStorage = new BankStorage(centralBank);
+    private final BankSystem bankSystem = new BankSystem(bankStorage);
+
+    public Person person(){
+         return new Person(bankKiosk());
+    }
+    public BankKiosk bankKiosk(){
```

Comment:
```suggestion
    private BankKiosk bankKiosk(){
```
Person()만 접근하고 있는 것으로 아는데, config내에서만 사용되는 메소드는 public으로 열어둘 필요가 없을 것 같습니다